### ISSUE TITLE
integration.cloud.clouds.test_digitalocean test failures

### ISSUE THREAD

**Issue by garethgreenaway** [saltstack/salt#53010]
2019.2 failed [salt-centos-7-py2-cloud](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py2-cloud/77/testReport/junit/integration.cloud.clouds.test_digitalocean/DigitalOceanTest/test_key_management), [salt-centos-7-py3-cloud](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py3-cloud/77/testReport/junit/integration.cloud.clouds.test_digitalocean/DigitalOceanTest/test_key_management)
---

'3b:16:bf:e4:8b:00:8b:b8:59:8c:a9:d3:f0:19:45:fa' not found in []

```

Traceback (most recent call last):
  File "/tmp/kitchen/testing/tests/integration/cloud/clouds/test_digitalocean.py", line 110, in test_key_management
    [i.strip() for i in _key]
AssertionError: '3b:16:bf:e4:8b:00:8b:b8:59:8c:a9:d3:f0:19:45:fa' not found in []
		
```

2019.2 failed [salt-centos-7-py2-cloud](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py2-cloud/77/testReport/junit/integration.cloud.clouds.test_digitalocean/DigitalOceanTest/test_instance), [salt-centos-7-py3-cloud](https://jenkinsci.saltstack.com/job/2019.2/job/salt-centos-7-py3-cloud/77/testReport/junit/integration.cloud.clouds.test_digitalocean/DigitalOceanTest/test_instance)
---

'CLOUD-TEST-1AKG0L' not found in []

```

Traceback (most recent call last):
  File "/tmp/kitchen/testing/tests/integration/cloud/clouds/test_digitalocean.py", line 145, in test_instance
    [i.strip() for i in self.run_cloud('-p digitalocean-test {0}'.format(INSTANCE_NAME), timeout=500)]
AssertionError: 'CLOUD-TEST-1AKG0L' not found in []
		
```

**Comment by KChandrashekhar** [comment]
Test is failing on 2019.2 branch and not on 2019.2.1 so it will be fixed in 2019.2.2 release

**Comment by Ch3LL** [comment]
i see this failing on 2019.2.1 here: https://jenkinsci.saltstack.com/job/2019.2.1/view/Cloud/job/salt-centos-7-py2-cloud/52/

**Comment by Ch3LL** [comment]
i have verified this is fixed for 2019.2.1 now. there was an issue with the key on digitaloceans servers. I had to manually remove.

looks like `integration.cloud.clouds.test_digitalocean.DigitalOceanTest.test_instance` test is still failing on 2019.2 so I will keep this open for 2019.2.2

**Comment by Ch3LL** [comment]
i'm seeing this fail again on 2019.2.1. Looks like the key is probably not getting deleted properly for some reason sometimes. Will look into it further
